src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const lastPage = Math.ceil(total / perPage);
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === lastPage;
+
+  return (
+    <ul className="pagination">
+      <li className={classNames('page-item', { disabled: isFirstPage })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={isFirstPage}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+        >
+          «
+        </a>
+      </li>
+
+      {getNumbers(1, lastPage).map(page => (
+        <li
+          className={classNames('page-item', {
+            active: currentPage === page,
+          })}
+          key={page}
+
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${page}`}
```

Comment:
Use `e.preventDefault()` to prevent the default behavior of the link when clicking on the pageLink.